welcome/index.adoc

```diff
@@ -20,10 +20,32 @@ and the xref:../release_notes/ocp-4-3-release-notes.adoc#ocp-4-3-release-notes[{
 endif::[]
 endif::[]
 
-ifdef::openshift-dedicated,openshift-online,openshift-aro[]
+ifdef::openshift-dedicated,openshift-online[]
 You can start with an **xref:../architecture/architecture.adoc#architecture-overview-architecture[Introduction to {product-title}]**.
 endif::[]
 
+ifdef::openshift-aro[]
+
+{product-title} is supported by Red Hat and Microsoft. Use the following table to navigate all the available resources related to {product-title}.
```

Comment:
You might say that documentation unique to the Azure Red Hat OpenShift service (e.g. how to create a cluster, how to get support) is generally to be found on the Microsoft Azure documentation site; documentation that is common to all OpenShift flavours is generally to be found on this documentation site?